### PR TITLE
APMRover2: Reduce the home position reset when disarm

### PR DESCRIPTION
**Pull request by khancyr**
This prevent to spam with home position change at 5Hz

### CODE REVIEW

File: APMrover2/APMrover2.cpp [hunk 1]
```diff
@@ -359,6 +359,13 @@ void Rover::one_second_loop(void)
 
     ins.set_raw_logging(should_log(MASK_LOG_IMU_RAW));
 
+    // update home position if soft armed and gps position has
```
Comment (gmorph) [inline APMrover2/APMrover2.cpp]: This should say "not soft armed"

**Comment by gmorph** [comment]
Thanks @khancyr - just make that small wording change in the comments and I'll pull this in.

Thanks, Grant.

File: APMrover2/APMrover2.cpp (hunk 1 above)
Comment (khancyr) [inline APMrover2/APMrover2.cpp]: done !

**Comment by khancyr** [comment]
@gmorph Change done !

**Pull request merged**